uk.co.piggz.amazfish.json

```diff
@@ -0,0 +1,152 @@
+{
+  "app-id": "uk.co.piggz.amazfish",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
```

Comment:
This is EOL.

Reply:
This is a blocker.

Reply:
I understand that EOL runtime is a problem. I have updated it to another 5.15 version. I don't see a list of EOLed SDKs or how long they will be supported.